wiki/Rules/th.md

```diff
@@ -0,0 +1,66 @@
+2. **อย่าเล่นขี้โกง** การใช้โปรแกรมภายนอกเพื่อที่จะได้ความได้เปรียบกว่าผู้เล่นคนอื่น ซึ่งรวมถึงการใช้ มาโคร (Macro), โปรแกรมช่วยเล็ง, การปรับความเร็ว และอย่างอื่นอีกมากมาย ถ้าโปรแกรมนั้นช่วยเล่นเกมในส่วนที่คุณควรจะทำ เช่นการกด, การเล็ง เป็นต้น มันไม่โอเคเลย
+3. **ทำดีต่อกันและกัน** การคุกคามหรือการทะเลาะกัน ไม่ควรนำมาอยู่ในชุมชนของ osu! เรามาที่นี้เพื่อที่จะมานั่งกดวงกลม, ตีกลอง, รับผลไม้ที่ตกลงมา, กดปุ่มมากได้ถึง 18ปุ่มภายในครั้งเดียว! ไม่ได้มาเพื่อทะเลาะกัน
+4. **อย่าทำตัวเป็นพวกน่ารำคาญ** ถ้ามีตอนไหนที่คุณคิดว่าตัวเองได้ทำผิดกฏข้อนี้ไปหรือเปล่า คุณน่าจะทำแหละ
+5. **เราเป็นชุมชนที่มีทุกวัย** ดังนั้นแปลว่า อะไรก็ตามที่เป็น 18+/[NSFW](https://en.wikipedia.org/wiki/Not_safe_for_work) เช่นเรื่องการเสพยา หรือเรื่องหื่นกามต่าง ๆ จะไม่ถูกต้อนรับที่นี่ จะมีข้อยกเว้นเฉพาะในเรื่องที่เหมาะสมเท่านั้น แต่ถ้าตามปกติแล้ว อย่าพูดถึงเรื่องพวกนี้
+6. **ถ้าเรื่องอะไรอยู่นอกเหนือกฏ ควรใช้ความคิดเบื้องต้นคิดเอง** ทีมงานดูแลลงโทษผู้ใช้ตามความเหมาะสม
+
+## กฏของแชทในเกม
+
+1. **ให้เกียรติซึ่งกันและกัน หลีกเลี่ยงการฟลัด และสแปม** การที่คนคนเดียวใช้แชทไปครึ่งจอภายในครั้งเดียวนั้นมันน่ารำคาญ ดังนั้นหลีกเลี่ยงการทำอย่างงั้นถ้าเป็นไปได้ คนอื่นก็อยากจะพูดคุยเหมือนกัน!
+2. **ปล่อยให้ผู้ดูแลจัดการปัญหาภายในแชท** คุณสามารถรายงานผู้ใช้คนอื่นได้โดยการใช้ [คำสั่ง `!report`](/wiki/Reporting_Bad_Behaviour) และพวกเขาจะจัดการที่เหลือเอง
+3. **ถ้าใครโดน Timed out, โดนเงียบ (Silenced), หรือโดนแบน (Restricted) ปล่อยมันไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบน กับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!
+4. **สนับสนุนการเล่นไม่โกง และอย่าไปให้ความสนใจกับพวกโกง หรือพวก toxic** น่าเสียดายที่มันมีคนพวกนี้อยู่ และการให้ความสนใจพวกเขานั้น จะทำให้เขาอยากทำต่อไป หรือชักชวนผู้อื่นให้ทำด้วย คุณสามารถรายงานได้โดยการใช้คำสั่ง `!report` หรือปุ่มรายงานบนหน้าโปรไฟล์ของพวกเขา และไม่ต้องไปสนใจเขา
+5. **ทำให้แชทสะอาด และเหมาะสมสำหรับสังคมทุกวัย หลีกเลี่ยงเนื้อหา NSFW ถ้าเป็นไปได้** ของพวก NSFW มันไม่ค่อยเหมาะกับชุมชนที่มีทุกวัย ทำอะไรก็ได้ที่คุณอยากทำในข้อความส่วนตัว (อย่างมีเหตุผล) แต่ทำให้แชทส่วนรวมสะอาดปลอดภัยจากเรื่อง NSFW สำหรับทุกคน
+6. **คิดก่อนจะส่งอะไรให้คนอื่น โดยเฉพาะสิ่งที่มีเนื้อหารบกวนใจ หรือสิ่งที่ทำคนอื่นไม่สบายใจ** ถ้าคุณต้องคิดซ้ำว่า สิ่งที่คุณจะส่งนั้นมันเหมาะสมไหม มันคงจะไม่เหมาะสมแหละ หลีกเลี่ยงสิ่งที่ เร็ว, ไฟกระพริบ (บางคนมีโรคเกี่ยวกับพวกนี้), เลือด, ภาพที่แสดงถึงการละเมิดต่าง ๆ
+7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฏหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฏหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่ค่อยจะโอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->
```

Comment:
```suggestion
7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฏหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฏหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่โอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->
```
จากต้นฉบับ "aren't okay" แบบว่า clear cut อ่ะ เปลี่ยนเป็น "ไม่โอเคเลย" ก็ได้นะ